frontend/src/views/pages/vehicleLists/VehicleList.vue

```diff
@@ -0,0 +1,322 @@
+        let routeData = this.$router.resolve({name: 'Vehicle Inventory', query: {vehicleSelected: vehicle._id}});
+        window.open(routeData.href, '_blank');
+      } else {
+        if (!['INPUT', 'LABEL'].includes(e.target.tagName)) {
+          this.check(vehicle)
+        }
+      }
+    },
+    fetchVehicleListDetails() {
+      axios({
+        method: "GET",
+        url: `${this.$store.state.api}/vehicle-list/${this.vehicleListId}`,
+        headers: {
+          Authorization: `Bearer ${this.$store.state.auth.token}`,
+        },
+      }).then((response) => {
+        this.vehicleList = response.data.payload;
+        this.notes = this.vehicleList.notes;
+        this.title = this.vehicleList.title;
+        this.editedTitle = this.title;
+
+        let tableItems = [];
+        this.vehicleList.vehicles.forEach((vehicle, index) => {
+          //Check if vehicle has properties
+          if (!this.delivered_bool) {
+            if (!vehicle.delivered) {
+              if (vehicle.properties != null)
+              { 
+                let properties = vehicle.properties;
+                properties._id = vehicle._id;
+                properties.vin = vehicle.vin;
+                properties.index = index;
+                if (vehicle.missing) {
+                  properties._classes = 'table-warning';
+                }
+                tableItems.push(properties);
+              }
+            }
+          }
+          if (this.delivered_bool) {
+            if (vehicle.delivered) {
+              if (vehicle.properties != null)
+              { 
+                let properties = vehicle.properties;
+                properties._id = vehicle._id;
+                properties.vin = vehicle.vin;
+                properties.index = index;
+                if (vehicle.missing) {
+                  properties._classes = 'table-warning';
+                }
+                tableItems.push(properties);
+              }
+            }
+          }
+        });
+        this.tableItems = tableItems;
+        this.fetchVehicleProperties();
+      }).catch((error) => {
+        console.log(error);
+      });
+    },
+    fetchVehicleProperties() {
+      axios({
+        method: "GET",
+        url: `${this.$store.state.api}/dealerships/${this.vehicleList.dealership}/vehicles/properties`,
+        headers: {
+          Authorization: `Bearer ${this.$store.state.auth.token}`,
+        },
+      }).then((response) => {
+        const payload = response.data.payload;
+        const fields = [];
+        fields.push({key: "vin", label: "VIN"});
+        payload.forEach((property) => {
+          if (property.visible) {
+            fields.push(property);
+          }
+        });
+        this.tableFields = fields;
+      }).catch((error) => {
+        console.log(error);
+        //this.$router.replace("/pages/404");
+      });
+    },
+    saveTitle() {
+      this.saveChanges();
+      this.updateTitle();
+      this.editingTitle = false;
```

Comment:
When the title is edited, the custom vehicle list table doesn't get updated all the time. I guess this could be a timing issue. Make use of the async and await function.

Reply:
Fixed in this [commit](https://github.com/purplesmurf1998/Autotracks/pull/62/commits/41dc75ec16d0d2dde8f4c9e25bc8f9d88178e111)